test/integration/targets/mysql_db/tasks/state_dump_import.yml

```diff
@@ -58,12 +73,71 @@
 - name: state dump/import - file name should exist
   file: name={{ db_file_name }} state=file
 
+- name: state dump with multiple databases in comma separated form.
+  mysql_db:
+    name: "{{ db_name }},{{ db_name2 }}"
+    state: dump
+    target: "{{ dump_file1 }}"
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result1
+
+- name: assert successful completion of dump operation (with multiple databases in comma separated form)
+  assert:
+    that:
+       - "dump_result1.changed == true"
+
+- name: state dump - dump file1 should exist
+  file: name={{ dump_file1 }} state=file
+
+- name: state dump with multiple databases in list form via check_mode
+  mysql_db:
+    name:
+      - "{{ db_name }}"
+      - "{{ db_name2 }}"
+    state: dump
+    target: "{{ dump_file2 }}"
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result
+  check_mode: yes
+
+- name: assert successful completion of dump operation (with multiple databases in list form) via check mode
+  assert:
+    that:
+       - "dump_result.changed == true"
+
+- name: database dump file2 should not exist
+  file: name={{ dump_file2 }} state=absent
```

Comment:
```suggestion
  file: name={{ dump_file2 }} state=absent
  register: result
  
- assert:
    that:
    - result.changed == false
```

Reply:
In this test you try to check that the dump file does not exist after check_mode execution of the module, right?
However, seems it just remove file if it exists, doesn't it?

From the official [documentation](https://docs.ansible.com/ansible/latest/modules/file_module.html)
> If absent, directories will be recursively deleted, and files or symlinks will be unlinked. Note that absent will not cause file to fail if the path does not exist as the state did not change.

Reply:
(and better not to use force-pushing if it is not necessary because we can't view changes in this case)